snap/epoch.go

```diff
@@ -342,6 +337,18 @@ func (z *uint32slice) UnmarshalJSON(bs []byte) error {
 	return nil
 }
 
+func isIncreasing(z []uint32) bool {
+	if len(z) == 0 {
```

Comment:
Since you are already doing the check:
```
if len(z) < 2 { return true }
```

Reply:
heh, good point. I'll tweak it in this PR if there are other related changes, otherwise I'll slide it in to the checker pr (upcoming).